CRM/Volunteer/BAO/Project.php

```diff
@@ -169,6 +169,40 @@ public static function getContactsByRelationship($projectId, $relationshipType)
   }
 
   /**
+   * Gets location of a project.
```

Comment:
While I like the idea of passing location data into the activity, CiviVolunteer [already](https://github.com/civicrm/org.civicrm.volunteer/blob/master/api/v3/VolunteerProject.php#L226) has [so](https://github.com/civicrm/org.civicrm.volunteer/blob/5c0b4049f0391ede6d89d82f8988c561122225a6/ang/volunteer/Projects.js#L31) many [kludges](https://github.com/civicrm/org.civicrm.volunteer/blob/2da10302f9c0a6242bd75a8d1844c03aee62862c/ang/volunteer/Project.js#L196) for [dealing](https://github.com/civicrm/org.civicrm.volunteer/blob/59534288f37c96202434c57adc73d2fa5dc7ba55/volunteer.php#L580) with [locations](https://github.com/civicrm/org.civicrm.volunteer/blob/5c0b4049f0391ede6d89d82f8988c561122225a6/ang/volunteer/Projects.html#L89) that [I'm](https://github.com/civicrm/org.civicrm.volunteer/blob/5c0b4049f0391ede6d89d82f8988c561122225a6/ang/volunteer/Projects.js#L103) loathe to add another. And that's not to mention the kludges I have in customer implementations! I think it's high time we modify api.VolunteerProject.get to return more usable values so we don't have to keep doing string gymnastics.

How do you feel about working on that together instead of adding yet another locations hack?